doc/lua_api.md

```diff
@@ -5269,6 +5270,12 @@ Utilities
       mod_storage_on_disk = true,
       -- "zstd" method for compress/decompress (5.7.0)
       compress_zstd = true,
+      -- Sound parameter tables support start_time (5.8.0)
+      sound_params_start_time = true,
+      -- New fields for set_physics_override: speed_climb, speed_crouch,
+      -- liquid_fluidity, liquid_fluidity_smooth, liquid_sink,
+      -- acceleration_default, acceleration_air (5.8.0)
+      new_physics_overrides = true,
```

Comment:
What will be the follow-up naming scheme when more are added? `new_new_physics_overrides`? 
Might a numeric value make more sense in this case? e.g. `physics_overrides_version`.

Reply:
How about `physics_overrides_5_8` then?

Reply:
I rather meant something like `physics_overrides_version = 2` which would then simply allow to increase for each new feature instead of adding new flags. Mods could still perform feature checks with `(minetest.features.physics_overrides_version or 1) >= 2` - at least for server-side support.

Client-side support might be needed in addition if modders want to ensure that the client actually supports the overrides.

Reply:
Done.